wafer/settings.py

```diff
@@ -148,6 +148,8 @@
     'wafer.sponsors',
     'wafer.pages',
     'wafer.tickets',
+    # Django isn't finding the overridden templates
+    'registration',
```

Comment:
From the description, I suspect this is related to #113 

I haven't looked at the django-registration code to see why this fix works, and it may not be something we want to rely on, but this change doesn't seem particularly harmful.